nevergrad/optimization/oneshot.py

```diff
@@ -65,18 +89,21 @@ class RandomSearchMaker(base.ParametrizedFamily):
 
     # pylint: disable=unused-argument
     def __init__(self, *, middle_point: bool = False, stupid: bool = False,
+                 quasi_opposite: str = "none",
```

Comment:
It's more "standard" Python to have: `quasi_opposite: Optional[str] = None`
also, I am not fond of the name, maybe "opposition_mode" would be better (currently it's like having a boolean named `true_false` :D )

Reply:
quasiopposite is standard terminology --- I follow the rest of your recommendation.

Reply:
Oh I add not seen you answered this, sorry. Is it standard to define one of the mode, or all the modes? It sounds like "one of them" (the "quasi" mode), that's why I was in favor of changing, I can change back, sorry :s